src/galaxy/SystemBody.cpp

```diff
@@ -217,6 +226,10 @@ double SystemBody::GetAtmAverageTemp(double altitude) const
 {
 	// temperature at height
 	const double lapseRate_L = CalcSurfaceGravity() / GetSpecificHeat(GetSuperType()); // deg/m
+	if (altitude > m_tropopause) {
+		return double(GetAverageTemp()) - lapseRate_L * m_tropopause;
+	}
```

Comment:
Note that this appears to set a constant atmospheric density above tropopause regardless of altitude. Is this actually physically correct?

If this is intended, this if-block isn't needed, as you can simply use `std::min(altitude, m_tropopause)` to replace the `altitude` scalar in the following line.